loader/src/platform/windows/main.cpp

```diff
@@ -138,11 +138,29 @@ void* mainTrampolineAddr;
 #include "gdTimestampMap.hpp"
 unsigned int gdTimestamp = 0;
 
+// In case the game is launched from a different directory through command line
+// this function will set the current working directory to the game's directory
+// to avoid the game crashing due to not being able to find the resources
+static void fixCWD() {
+    WCHAR cwd[MAX_PATH];
```

Comment:
we in geode refuse to use c related stuff as much as we can, and for good reason, using std::array here would be much better and would fix that bug below